browser/ui/views/overlay/brave_video_overlay_window_views.cc

```diff
@@ -235,12 +249,20 @@ void BraveVideoOverlayWindowViews::OnUpdateControlsBounds() {
                          close_controls_view_->width(),
                      kTopControlSpacing - close_button_insets.top()});
 
+  fullscreen_button_->SetSize(close_button_size);
+  fullscreen_button_->SetPosition(
+      {close_controls_view_->origin().x() -
+           (kTopControlSpacing - close_button_insets.left() -
+            fullscreen_button_->GetInsets().right()) -
+           fullscreen_button_->size().width(),
+       close_controls_view_->origin().y()});
+
   if (back_to_tab_label_button_) {
```

Comment:
It could be separated issue - it would be good to have tooltip for back to tab button.

Reply:
Oh, didn't know that it's cleaed when we call SetText({}) below. Thanks.